extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -120,6 +131,32 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
     if (!this.isOnActivePage_() || !isOpen) {
       return;
     }
+
+    if (document.activeElement.tagName === 'AMP-STORY-SHOPPING-TAG') {
```

Comment:
The active element is the last element that was in focus, so it would be the shopping tag (in the case of shopping tag clicks), or the body tag (in the case of the cta button being clicked)

Reply:
Given the discussion in the meeting here, it should be possible to do this without the branch statement. I'll rewrite it without this check.
https://docs.google.com/document/d/1YgLIMNDbmUq-rBw4D0FGbpts3xurg0hApKQOC-aVIZo/edit#heading=h.7zq9yzwl3rd